### ISSUE TITLE
LeBlender resolver credit?

### ISSUE THREAD

**Issue by leekelleher** [umbraco/Umbraco.Courier.Contrib#10]
@clausjensen Who developed the LeBlender grid cell resolver?
Was it from here: https://github.com/Lecoati/LeBlender/pull/30?

Just trying to tie up any loose ends with the credits.

**Comment by clausjensen** [comment]
@leekelleher I ended up redoing it myself ... I helped @hfloyd a while back when she was creating hers and wanted to just pull that in ... after going through it though, it seemed like there was both unused code, some unnecessary parts and some manual json object manipulation that I felt had potential for breaking stuff. 
I think @mattmorrisonsolidstudios changed some things and cleaned it up a bit, but to follow the "concepts" of the other resolvers I just redid it based on how the Vorto and Nested Content resolvers was built. I figured it would be better to add in whatever null and empty checks I may have missed from their implementations afterwards.

You're welcome to credit them - @hfloyd was your initial work based on anything that isn't already credited here .. I seem to remember you just started out with the DocTypeGridResolver and went from there... but it's a long time ago so I might be wrong?

**Comment by leekelleher** [comment]
I think it's cool to give credit to all who've helped get it this far... _good karma and all that!_ ... and we should include yourself on that too. :metal: ❤️ 

I'll send a PR later... and then draw a line under this part of the credits.

**Comment by hfloyd** [comment]
@clausjensen & @leekelleher - I used the NestedContent DR and any others I could find as examples for the LeBlender one. Thanks, @clausjensen for cleaning it up - I know it was a mess because I tried a hundred things while struggling to get around what ended up being a bug in Courier source (related to not being able to assign the manipulated data value back to the object). When it was finally working, I didn't have any more time to devote to it. :-)
